cache/remotecache/registry/registry.go

```diff
@@ -13,7 +13,6 @@ import (
 	"github.com/moby/buildkit/util/resolver"
 	"github.com/moby/buildkit/util/resolver/limited"
 	digest "github.com/opencontainers/go-digest"
-	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 	specs "github.com/opencontainers/image-spec/specs-go/v1"
```

Comment:
We should use a consistent alias for this import throughout the project; looks like it's currently a mix of `specs` and `ocispecs`;

<img width="478" alt="Screenshot 2021-07-26 at 09 47 20" src="https://user-images.githubusercontent.com/1804568/126952481-6b2ee9fa-c2da-472e-88e3-355af135c531.png">

Reply:
see https://github.com/moby/buildkit/pull/2277#issuecomment-886579329